frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useTable } from 'react-table';
+import { Column } from 'common/interfaces';
+import styles from './styles.module.scss';
+
+interface IProps {
+  columns: Column[],
+  data: Record<string, unknown>[],
```

Comment:
```suggestion
  data: CustomRecord,
```
from shared package

Reply:
In user which is transferred to table there is a geoposition which is not

Reply:
Remove geoposition from interface for now